mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/logging/impl/Log.java

```diff
@@ -101,4 +102,28 @@
 
 	@Message(id = 1228, value = "Query must be executed using 'getResultList()' or 'getSingleResult()' method: %s")
 	HibernateException readQueryMustBeExecutedViaGetResultList(MongoDBQueryDescriptor queryDescriptor);
+
+	@Message(id = 1229, value = "Unable to create index %2$s on collection %1$s")
+	HibernateException unableToCreateIndex(String collection, String indexName, @Cause Exception e);
+
+	@LogMessage(level = ERROR)
+	@Message(id = 1230, value = "Unable to create text index %2$s on collection %1$s. A text index named %3$s already exists and MongoDB only supports one text index per collection.")
+	void unableToCreateTextIndex(String collection, String newIndexName, String existingIndexName);
```

Comment:
**A** text index already exists in the db for this collection. It might be a totally different index or an index with the same name but a different configuration. The issue is that only one text index per collection is supported and MongoDB silently ignores all the text index creation orders if there is already one for the collection.